test/Distributed/Runtime/distributed_actor_dynamic_remote_func.swift

```diff
@@ -0,0 +1,92 @@
+// RUN: %target-run-simple-swift(-Xfrontend -enable-experimental-distributed -parse-as-library) | %FileCheck %s
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+// REQUIRES: distributed
+
+// UNSUPPORTED: use_os_stdlib
+// UNSUPPORTED: back_deployment_runtime
+
+import _Distributed
+
+@available(SwiftStdlib 5.5, *)
+distributed actor LocalWorker {
+  distributed func function() async throws -> String {
+    "Local"
+  }
+
+//  func function(name: String) async throws -> String {
+//    return name
+//  }
+}
+
+@available(SwiftStdlib 5.5, *)
+extension LocalWorker {
+  @_dynamicReplacement(for: _remote_function())
```

Comment:
This is awesome, but I wonder if we'll prefer `@_distributedRemoteReplacement(for: function())` instead, and we'd require such function to be `nonisolated` (the current shape is unsafe wrt the properties of the actor)